specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimapis.json

```diff
@@ -0,0 +1,1878 @@
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ApiUpdateContract"
+            },
+            "description": "API Update Contract parameters."
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": true,
+            "description": "ETag of the API entity. ETag should match the current entity state in the header response of the GET request or it should be * for unconditional update.",
+            "type": "string"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The API was successfully updated."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Apis"
+        ],
+        "operationId": "Api_Delete",
+        "description": "Deletes the specified API of the API Management service instance.",
+        "x-ms-examples": {
+          "ApiManagementDeleteApis": {
+            "$ref": "./examples/ApiManagementDeleteApis.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "required": true,
+            "description": "ETag of the API Entity. ETag should match the current entity state from the header response of the GET request or it should be * for unconditional update.",
+            "type": "string"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "The API was successfully deleted."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/apis/{apiId}/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "ApiOperation_ListByApi",
+        "description": "Lists a collection of the operations for the specified API.",
+        "x-ms-examples": {
+          "ApiManagementListApisOperations": {
+            "$ref": "./examples/ApiManagementListApisOperations.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiIdParameter"
+          },
+          {
+            "name": "$filter",
```

Comment:
>"name": "$filter", [](start = 12, length = 18)

Same here, if `filter` is the same in all operations, whould be better to move it to paramaters section

Reply:
$filter changes based on contract. it is unique description for each Path.

---
In reply to: [163343977](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163343977) [](ancestors = 163343977)